_posts/2021-06-16-pruning-the-oldest-issues.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Pruning the oldest issues - June 2021"
+date: 2021-06-16
+author: Jan Ainali
+type: blogpost
+excerpt: Another effort to reduce our Standard for Public Code backlog
+categories:
+  - Community call
+---
+
+# Pruning the oldest issues - June 2021
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
+* [Felix Faassen](https://publiccode.net/who-we-are/team/felix-faassen.html)
+
+## Log
+
+Following the [our new process](/news/2021/06/14/pruning-our-backlog.md) we continue reviewing our oldest Standard for Public Code issues. These were the ones we got through this time:
+
+- [Bundle policy and code: invite policy makers to contribute their own work in their own formats](https://github.com/publiccodenet/standard/issues/62) - session to work on this was calendered on June 14
+- [Template: Reference back to Foundations' core value propositions](https://github.com/publiccodenet/standard/issues/167) - closed after no new comments recieved since last session
+- [Add section on relation to coreinfrastrcture best-practice-badge](https://github.com/publiccodenet/standard/issues/200) - follow up in a future session
+- [Template: reorder the headers](https://github.com/publiccodenet/standard/issues/135) - closed after no new comments recieved since last session
```

Comment:
```suggestion
- [Template: reorder the headers](https://github.com/publiccodenet/standard/issues/135) - closed after no new comments received since last session
```